products/sle/main.pm

```diff
@@ -957,40 +957,20 @@ elsif (have_scc_repos()) {
     }
 }
 elsif (get_var('HPC')) {
-    if (check_var('HPC', 'install')) {
-        load_boot_tests();
-        load_inst_tests();
-        load_reboot_tests();
+    if (get_var('CLUSTER_NODES')) {
+        load_cluster_boot();
     }
     else {
-        if (get_var('CLUSTER_NODES')) {
-            load_cluster_boot();
-        }
-        else {
-            loadtest 'boot/boot_to_desktop';
-            loadtest 'hpc/setup_network' if check_var('NICTYPE', 'tap');
-        }
-        loadtest 'hpc/enable_in_zypper' if (sle_version_at_least('15'));
-        if (check_var('HPC', 'conman')) {
-            loadtest 'hpc/conman';
-        }
-        elsif (check_var('HPC', 'powerman')) {
-            loadtest 'console/hostname';
-            loadtest 'hpc/powerman';
-        }
-        elsif (check_var('HPC', 'hwloc') && sle_version_at_least('12-SP2')) {
-            loadtest 'console/hwloc_testsuite';
-        }
-        else {
-            loadtest 'console/install_all_from_repository' if (get_var('INSTALL_ALL_REPO'));
-            loadtest 'console/install_single_package'      if (get_var('PACKAGETOINSTALL'));
-
-            # load hpc multimachine scenario based on value of HPC variable
-            # e.g 'hpc/$testsuite_[master|slave].pm'
-            my $hpc_mm_scenario = get_var('HPC');
-            loadtest "hpc/$hpc_mm_scenario" if $hpc_mm_scenario ne '1';
-        }
+        loadtest 'boot/boot_to_desktop';
```

Comment:
while you are at it, could you please try to replace this by a call to `boot_hdd_image`? This is the way it should be done to support proper booting for all archs/machines

Reply:
boot_hdd_image requires to use uefi_bootmenu , why I need it ?

Reply:
see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4631 for that